lib/trino-orc/src/test/java/io/trino/orc/OrcTester.java

```diff
@@ -960,33 +969,77 @@ private static List<Object> decodeRecordReaderStruct(Type type, List<?> fields)
         return newFields;
     }
 
-    public static DataSize writeOrcColumnHive(File outputFile, Format format, CompressionKind compression, Type type, Iterator<?> values)
+    public static void writeOrcColumnHive(File outputFile, Format format, CompressionKind compression, Type type, Iterator<?> values)
             throws Exception
     {
         RecordWriter recordWriter = createOrcRecordWriter(outputFile, format, compression, type);
-        return writeOrcFileColumnHive(outputFile, recordWriter, type, values);
+        writeOrcColumnHive(recordWriter, type, values);
     }
 
-    public static DataSize writeOrcFileColumnHive(File outputFile, RecordWriter recordWriter, Type type, Iterator<?> values)
+    public static void writeOrcColumnHive(RecordWriter recordWriter, Type type, Iterator<?> values)
             throws Exception
     {
-        SettableStructObjectInspector objectInspector = createSettableStructObjectInspector("test", type);
+        StandardStructObjectInspector objectInspector = getStandardStructObjectInspector(ImmutableList.of("test"), ImmutableList.of(getJavaObjectInspector(type)));
+        writeOrcColumnsHive(
+                recordWriter,
+                objectInspector,
+                ImmutableList.of(type),
+                StreamSupport.stream(
+                        Spliterators.spliteratorUnknownSize(values, Spliterator.ORDERED), false)
+                        .map(value -> (Function<Integer, Object>) (fieldIndex) -> value)
+                        .iterator());
+    }
+
+    public static void writeOrcColumnsHiveFile(
+            File outputFile,
+            OrcTester.Format format,
+            CompressionKind compression,
+            List<String> names,
+            List<Type> types,
+            Iterator<Function<Integer, Object>> values)
+            throws Exception
```

Comment:
No change is requested. Throwing `Exception` is fine in a test method, but this is a helper method, not a test method.

Reply:
Actually it can be included (and it's included in other tests which didn't changed within PR) to test method, so it can be considered as a part of test. There are multiply exceptions are  throwing from:
```
Writable record = serializer.serialize(row, objectInspector); -> throws SerDeException
...
recordWriter.write(record); -> throws IOException
...
recordWriter.close(false); -> throws IOException;
```
do you propose wrap those to runtime exception?

Reply:
> are you proposing wrap those to runtime exception?

No. The suggestion was throwing those exceptions as-is.